instrumentation/restlet/restlet-1.0/library/src/main/java/io/opentelemetry/instrumentation/restlet/v1_0/RestletTracingBuilder.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.restlet.v1_0;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanStatusExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanStatusExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetAttributesExtractor;
+import java.util.ArrayList;
+import java.util.List;
+import org.restlet.data.Request;
+import org.restlet.data.Response;
+
+public final class RestletTracingBuilder {
+
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.restlet-1.0";
+
+  private final OpenTelemetry openTelemetry;
+  private final List<AttributesExtractor<Request, Response>> additionalExtractors =
+      new ArrayList<>();
+
+  RestletTracingBuilder(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  public RestletTracingBuilder addAttributesExtractor(
+      AttributesExtractor<Request, Response> attributesExtractor) {
+    additionalExtractors.add(attributesExtractor);
+    return this;
+  }
+
+  public RestletTracing build() {
+    HttpAttributesExtractor<Request, Response> httpAttributesExtractor =
+        new RestletHttpAttributesExtractor();
+    SpanNameExtractor<Request> spanNameExtractor =
+        HttpSpanNameExtractor.create(httpAttributesExtractor);
+    SpanStatusExtractor<Request, Response> spanStatusExtractor =
+        HttpSpanStatusExtractor.create(httpAttributesExtractor);
+    NetAttributesExtractor<Request, Response> netAttributesExtractor =
+        new RestletNetAttributesExtractor();
+
+    Instrumenter<Request, Response> instrumenter =
+        Instrumenter.<Request, Response>newBuilder(
+                openTelemetry, INSTRUMENTATION_NAME, spanNameExtractor)
+            .setSpanStatusExtractor(spanStatusExtractor)
+            .addAttributesExtractor(httpAttributesExtractor)
+            .addAttributesExtractor(netAttributesExtractor)
+            .addAttributesExtractors(additionalExtractors)
+            .newServerInstrumenter(RestletHeadersGetter.GETTER);
```

Comment:
You don't need the `GETTER` singleton, you can just create a new instance:
```suggestion
            .newServerInstrumenter(new RestletHeadersGetter());
```
Those setter/getter singletons are a mostly remnant from the BaseTracer API